kloppy/domain/models/event.py

```diff
@@ -423,6 +469,30 @@ def get_qualifier_value(self, qualifier_type: Type[Qualifier]):
                     return qualifier.value
         return None
 
+    def get_qualifier_values(self, qualifier_type: Type[Qualifier]):
+        """
+        Returns all Qualifiers of a certain type, or None if qualifier is not present.
+
+        Arguments:
+            qualifier_type: one of the following QualifierTypes: [`SetPieceQualifier`][kloppy.domain.models.event.SetPieceQualifier]
+                [`BodyPartQualifier`][kloppy.domain.models.event.BodyPartQualifier] [`PassQualifier`][kloppy.domain.models.event.PassQualifier]
+
+        Examples:
+            >>> from kloppy.domain import SetPieceQualifier
+            >>> pass_event.get_qualifier_value(SetPieceQualifier)
+            <SetPieceType.GOAL_KICK: 'GOAL_KICK'>
+        """
+        qualifiers = []
+        if self.qualifiers:
+            for qualifier in self.qualifiers:
+                if isinstance(qualifier, qualifier_type):
+                    qualifiers.append(qualifier)
+
+            if qualifiers:
+                return qualifiers
+
+        return None
```

Comment:
I would expect this function to always return `List` type. 

So always `return qualifiers`